frontend/pages/component/utils.tsx

```diff
@@ -1,3 +1,3 @@
-export function classNames(...classes: string[]): string {
+export default function classNames(...classes: string[]): string {
     return classes.filter(Boolean).join(' ');
 }
```

Comment:
You should remove the `default`  or rename the file, `utils.tsx` -> `classNames.tsx` for instance